valohai_yaml/schema/pipeline.yaml

```diff
@@ -38,6 +38,13 @@ properties:
           required:
             - source
             - target
+  allow-reuse:
```

Comment:
Would it make sense to call it `node-reuse` with true/false values?

Reply:
Or `step-reuse`

Reply:
we called it `allow_reuse` in roi's code too so we could keep it for consistency

Reply:
> we called it allow_reuse in roi's code 

Do you mean in the internal code or in the user-facing UI?

What we call it internally doesn't matter to the end users. In the UI we had some preamble text explaining what the feature does and how it works. In `valohai.yaml`, the terminology we use should be self-explanatory, and `allow-reuse` isn't descriptive enough of what the feature does.

Reply:
sure, let's use `node-reuse`/`step-reuse` as you suggested. However, IMO, from user's perspective, it doens't matter that much because whichever keyword we use, it wouldn't be self-explanatory enough for this feature, user will have to look for the description and/or documentation anyway to be able to understand what the keyword does.

Reply:
TBH, I liked `allow-reuse` more. `node-reuse` is inaccurate, since it does not reuse nodes, but executions that would be executed by some nodes. `reuse-executions` smells like it would be most accurate for what we have now.